config/initializers/omniauth.rb

```diff
@@ -16,6 +16,9 @@
 Rails.application.config.omniauth_google = ENV['GOOGLE_OAUTH2_ID'].present? && ENV['GOOGLE_OAUTH2_SECRET'].present?
 Rails.application.config.omniauth_office365 = ENV['OFFICE365_KEY'].present? &&
                                               ENV['OFFICE365_SECRET'].present?
+Rails.application.config.omniauth_saml = ENV['SAML_ISSUER'].present? && ENV['SAML_IDP_URL'].present? &&
+                                         ENV['SAML_IDP_CERT_FINGERPRINT'].present?
+
```

Comment:
Increase the chances of getting this commited by removing this extra empty line that rubocop complains about...

Reply:
I hope this increases the chances by at least 100 %